### DISCUSSION TITLE
Integration 'prochains_rer' not found.

### DISCUSSION THREAD

**Discussion by DanBendavid** [niicojs/ha_prochains_rer#1] [Q&A]
Hi Niicojs

I am very happy to find your home assistant custom component but i am not able to install it.....

Platform error sensor.prochains_rer - Integration 'prochains_rer' not found.

I am with HA for more than 2year now... i really need your help

Thanks

**Comment by niicojs** [comment]
Could you send me a screenshot of `/config/custom_components\prochains_rer` ?

It should look like this:
![image](https://user-images.githubusercontent.com/5653050/144186449-9923e46b-f545-4ad6-aec2-fe79df08373f.png)

**Comment by DanBendavid** [comment]
Hi, the py_cache_ is not generated. 
![image](https://user-images.githubusercontent.com/38190460/144204747-3d883912-cecd-4360-b998-d1c43a5f07aa.png)

Could you send me your manifest ?

**Reply by niicojs** [reply]
The manifest should be exactly the one on github.
If you look at your log files, do you see anything suspect?

In supervisor / System / Core

**Comment by DanBendavid** [comment]
2021-12-01 09:58:42 ERROR (SyncWorker_0) [homeassistant.loader] The custom integration 'prochains_rer' does not have a version key in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details

**Comment by niicojs** [comment]
Ok, I've uploaded a new manifest to github that should work. Keep me posted!

**Comment by DanBendavid** [comment]
Well it is functioning very well. I have one strange minor Bug with a specific terminus code 87534131 not available in gare.json

**Comment by DanBendavid** [comment]
{ "uic": "87534131", "nom": "ERMONT EAUBONNE" },
You can add this line into gares.json
